apis/vertex_resource.py

```
@@ -65,23 +78,76 @@ def post(self):
+        """
+        Endpoint to handle Google's Vertex/Gemini.
+        Receives a message from the user, processes it, and returns a stream response from the model.
+        """
+        data = request.json
+        try:
+            if data.get('stream') == "True":
+                data['stream'] = True  # convert to boolean
+            model = data.get('model')
+            app.logger.info(f'handling gemini request using {model}')
+            chat_model = genai.GenerativeModel(model)
+            parameters = {
```

Ruff (0.13.3)

Local variable `parameters` is assigned to but never used

Remove assignment to unused variable `parameters`

(F841)